doc/advanced-configuration/outgoing-connections.md

```diff
@@ -501,3 +502,67 @@ below the limit.
    {max_worker_queue_len, 100}]}
 ]}.
 ```
+
+## LDAP connection setup
+
+To configure a pool of connections to an LDAP server, use the following syntax:
+
+```erlang
+{ldap, Host, Tag, PoolOptions, ConnectionOptions}
+```
+
+### Connection options
+
+The following options can be specify in the `ConnectionOptions` list:
```

Comment:
"can be specify" -> "can be specified"

Reply:
Ok, I wonder where I copy-pasted this from, I will check.